util/execdetails.go

```diff
@@ -663,6 +670,17 @@ func (td *TimeDetail) String() string {
 	return buf.String()
 }
 
+// Merge merges the time detail into itself.
+func (td *TimeDetail) Merge(detail *TimeDetail) {
```

Comment:
Where is this used?

Reply:
To be used here https://github.com/pingcap/tidb/blob/master/pkg/distsql/select_result.go#L531 to record the `TimeDetail` information like the `ScanDetail`.